.github/CONTRIBUTING.md

```diff
@@ -82,7 +82,7 @@ Any new components or updates to existing components require the following:
   - `npm run build`
 - Documentation builds locally.
   - `npm run storybook:build`
-  
+
 ### How to Submit
 
```

Comment:
If your PR contains visual UI changes, please add images for references like `BEFORE` and `AFTER` to show whats the change's impact. Also add steps to reproduce if it is necessary.

Reply:
Something like this?

Reply:
> If your PR contains visual UI changes, please add images for references like `BEFORE` and `AFTER` to show whats the change's impact. Also add steps to reproduce if it is necessary.

Since we have a section for images in the PR template I added it there, think it will be a bit more visible.